crates/rolldown_error/src/build_error/error_constructors.rs

```diff
@@ -261,6 +263,19 @@ impl BuildDiagnostic {
     Self::new_inner(AssignToImport { filename, source, span, name })
   }
 
+  #[allow(clippy::cast_possible_truncation)]
+  pub fn json_parse(
+    filename: ArcStr,
+    source: ArcStr,
+    line: usize,
+    column: usize,
+    message: ArcStr,
+  ) -> Self {
+    let start_offset = line_column_to_byte_offset(source.as_str(), line - 1, column - 1);
```

Comment:
serde_json err is one-based https://docs.rs/serde_json/1.0.132/serde_json/struct.Error.html#method.column

Reply:
Would it be better to comment in code directly?

Reply:
ok